packages/types/src/vscode.ts

```diff
@@ -80,7 +83,10 @@ export const languages = [
 	"pl",
 	"pt-BR",
 	"ru",
+	"sv", // kilocode_change
+	"th", // kilocode_change
 	"tr",
+	"uk", // kilocode_change
```

Comment:
to prevent mergeconflicts, i elsewhere did it like this:

```
const kiloLanguages = ['ca', 'cas']
export const languages = ['de', 'en', ...kiloLanguages]
```
That way we have less change of conflict I think

Reply:
Are conflict really that likely here? I think it would make eventual upstreaming more difficult.